main.md

```diff
@@ -686,9 +686,10 @@ and revealing the claim names does not provide any additional information.
 
 ## Unlinkability 
 
-It is also important to note that this format enables selective disclosure of claims, but
-in itself it does not achieve unlinkability of the subject of an SD-SWT.
-
+Colluding issuer/verifier or verifier/verifier pairs could learn undisclosed holder claims by linking 
```

Comment:
Learning of undisclosed holder claims does not sound like an unlinkability issue. 
In the case of (verifier1+verifier2)-holder1-unlinkability, it is the linking of holder1's presentations (protocol executions) at verifier1 and verifier2. This is the failure of the pairwise pseudonymity property. 

If we want to talk about the uncovering of claims by colluding issuer and verifier, then we probably want to have another section on it, and IMHO, we should have one.

Reply:
I made a suggestion wrt unlinkability text. I agree learning undisclosed holder claims is a separate issue that we should probably have a text about in security considerations, outside of this PR.

Reply:
I mean claim "values" here, not their type. Linkability can be established on the basis of any correlatable information, which could be any unique value (serial number, signature value, a claim value).